src/utils/collection/index.ts

```diff
@@ -77,7 +77,7 @@ export function getSelectedCollection(
     return collections.find((collection) => collection.id === collectionID);
 }
 
-export function isSharedCollection(
+export function isIncomingSharedCollection(
```

Comment:
we can remove the three `isXYZCollection` function now ?as it is not used any where ?